op/k8s/kubelet_boot.go

```diff
@@ -364,7 +364,6 @@ func KubeletServiceParams(n *cke.Node, params cke.KubeletParams) cke.ServicePara
 		"--kubeconfig=/etc/kubernetes/kubelet/kubeconfig",
 		"--hostname-override=" + n.Nodename(),
 	}
-	args = append(args, "--container-runtime=remote")
```

Comment:
`--container-runtime` is removed.
https://github.com/kubernetes/kubernetes/pull/114017